### PR TITLE
use api delete post

### PR DESCRIPTION
**Pull request by ghost**
## Description
Use api delete post
## Reference Requirements
- Task ID: TI-1

## Reference Isuees
#{issue_id}

## Migrations
YES | NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: public/js/user/showPostOfProduct.js [hunk 1]
```diff
@@ -32,9 +36,14 @@ function getData(url, typePost) {
             }
           }
         }
-        html+='<div class="review-ratting">\
+        let showAction = '';
```
Comment (at-thaitran) [inline public/js/user/showPostOfProduct.js]: `let->var`

File: public/js/user/showPostOfProduct.js [hunk 2]
```diff
@@ -2,12 +2,16 @@ const ENABLE = 1;
 const TYPE_REVIEW = 2;
 const TYPE_COMMENT = 1;
 var path = document.location.pathname;
+var data_user = JSON.parse(localStorage.getItem('data'));
+url_delete = '/api/posts/';
```
Comment (at-thaitran) [inline public/js/user/showPostOfProduct.js]: var url...

**Review by ghost — COMMENTED** [review]
please check my comments

**Review by ghost — APPROVED** [review]
LGTM [:+1:](http://lgtm.in/u/0aNBSIYwn)
[LGTM](http://lgtm.in/i/0aNBSIYwn)

**Pull request merged**